### DISCUSSION TITLE
Is it possible to have different colours for stdout vs stderr?

### DISCUSSION THREAD

**Discussion by acheronfail** [kovidgoyal/kitty#7128] [Q&A] [answered]
I don't even know if this is possible for a terminal emulator. Does it know a program's stdout vs stderr, or is it only a psuedo-tty or something and the terminal emulator can't tell the difference?

Either way, I thought this could be a nice feature: _be able to theme stdout slightly differently to stderr_. If one could provide a set of theme colours for stdout vs stderr, it could be really easy at a glance to grok what programs are outputting to.

I was thinking that this would be nice to have across the board, rather than solutions which require intercepting write calls, or piping each stdout/stderr manually for every single command.

**Comment by kovidgoyal** [comment, marked as answer]
No stdout and stderr all come to the terminal on the same pipe.

**Reply by acheronfail** [reply]
I thought so. Thanks for confirming it! :heart:
